src/test/components/MenuButtons.test.js

```diff
@@ -469,16 +485,25 @@ describe('app/MenuButtons', function() {
   });
 
   describe('Full View Button', function() {
+    function setupForFullViewButton() {
+      return setup(storeWithProfile(createSimpleProfile().profile));
+    }
```

Comment:
Some going back and forth in this patch sequence, sorry about that 😅. Here we implement a new simple setup function for this test group after all.